md/release-notes.md

```diff
@@ -42,6 +42,12 @@ To be documented
 
 ### API removals
 
+#### BDM Api removal
+Since 7.0, BDM APIs and model classes in `com.bonitasoft.engine.bdm.*` packages where deprecated in favor of the Community version in `org.bonitasoft.engine.bdm.*` packages.  
```

Comment:
"have been" instead of "where"

Reply:
Nop. I say that *Since 7.0*, meaning that they were deprecated already, this is not new => simple past or present.
=> will change to **are**